io/tests/src/test/java/org/wildfly/extension/io/IOSubsystemTransformerTestCase.java

```diff
@@ -53,7 +52,7 @@ public IOSubsystemTransformerTestCase() {
 
     @Override
     protected String getSubsystemXml() throws IOException {
-        return readResource("io-2.0-transformer.xml");
+        return readResource("eap-7.1.0-transformer.xml");
```

Comment:
why did you rename the XML files using EAP versions instead of the subsystem version?